server/GameController.js

```diff
@@ -206,8 +164,11 @@ module.exports = function(data) {
 
 			data = {
 				players: players,
-				roundQuestion: round.question,
-				round: roundCount,
+				roundSubmissionData: round.getRoundSubmissionData(),
```

Comment:
can you just return the round object? Or have a round to Object method that returns all neccessary fields?